### ISSUE TITLE
char '#' without autocomplete 

### ISSUE THREAD

**Issue by DevTareq** [podio/jquery-mentions-input#102]
Hi, 

I used the '#' and its working fine, but what i am trying to do is letting the user to create a hashtag if he didn't find what he is looking for in the autocomplete list , is this doable? 

so the hashtag will be highlighted while the user still typing his custom hashtag.

Thanks,

**Comment by cyrusstoller** [comment]
+1

**Comment by djhvscf** [comment]
Yes!! Sounds great.. Remember that any pull request is welcome!! :+1:
